CHANGELOG.md

```diff
@@ -21,11 +30,19 @@ Mark  items as `Added`, `Changed`, `Fixed`, `Removed`, `Untested Features`, `Upc
 
 * Added multi-tenancy support.
 * Added support for default workflow with form.
+* Added API support for `draft` feature.
+* Added API support for `Form Adapter`.
 
 **forms-flow-bpm**
 
 * Added BPM_BASE_URL as the base url for camunda without /camunda- New property included with multitenancy.
```

Comment:
This can be removed as it is no longer needed.
Also need to add the change of variable name CAMUNDA_API_URI to BPM_API_URL